README.md

```diff
@@ -34,7 +34,7 @@ The following parameters are optional for your config file:
 | `replica_set` | string | name of replica set |
 | `ssl` | Boolean | can be set to true to connect using ssl, default false |
 | `verify_mode` | Boolean | Default SSL verify mode, default true |
-| `include_schema_in_destination_stream_name` | Boolean | forces the stream names to take the form `<database_name>_<collection_name>` instead of `<collection_name>`|
+| `include_schemas_in_destination_stream_name` | Boolean | forces the stream names to take the form `<database_name>_<collection_name>` instead of `<collection_name>`|
```

Comment:
what is the default value of this?

PPW targets expecting the schemas names and I think we need to use '-' instead of '_' otherwise [target-snowflake:here](https://github.com/transferwise/pipelinewise-target-snowflake/blob/785626918310106fa85a23a4e719ec8e7e58fbb0/target_snowflake/db_sync.py#L172) cannot decode it.

Reply:
Default value is False, check it out [here](https://github.com/transferwise/pipelinewise-tap-mongodb/blob/adcb0effb90c6aa5502d564be378f0180e5e22d8/tap_mongodb/__init__.py#L259).

Actually, now that I'm working on the import stuff in pipelinewise, I'm gonna leave it as is, False and use the table name (collection) as the stream name.